docs/longhornctl.md

```
@@ -26,4 +26,4 @@ CLI for Longhorn troubleshooting and operations.
 * [longhornctl trim](longhornctl_trim.md)	 - Longhorn trimming operations
 * [longhornctl version](longhornctl_version.md)	 - Print longhornctl version
 
-###### Auto generated by spf13/cobra on 13-Oct-2024
+###### Auto generated by spf13/cobra on 26-Nov-2024
```

Markdownlint (0.35.0)

Expected: h4; Actual: h6
Heading levels should only increment by one level at a time

(MD001, heading-increment)